### PR TITLE
Hacktober

### PR DESCRIPTION
**Pull request by felipedotcom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
-# CyBear Jinni Smart Device
+<h1 align="center">CyBear Jinni Smart Device</h1>
 
-Welcome
+[![CyBear Jinni](https://user-images.githubusercontent.com/9304740/94843279-24a49900-0425-11eb-83f3-87e8ba40b1dc.png)](https://github.com/CyBear-Jinni/CBJ_Smart-Home)
+
+<div align="center">
+
+![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg) [![style: lint](https://img.shields.io/badge/lint-1.3.0-blue)](https://pub.dev/packages/lint) [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0) [![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://en.wikipedia.org/wiki/Open_source)
```
Comment (guyluz11) [inline README.md:7]: dart CI

Will not work as a link, change it to 

[dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/actions)

Comment (felipedotcom) [reply README.md]: good now?

**Comment by guyluz11** [comment]
Thanks for the pull request @felipedotcom

**Comment by guyluz11** [comment]
If you would like to help us make the world a better place a ⭐star🌟 to the project will help.

Happy hacktoberfest :)

**Pull request merged**